rules/community/fleet/fleet_bad_action.py

```diff
@@ -0,0 +1,26 @@
+from stream_alert.shared.rule import rule
+from stream_alert.shared.rule import disable
+from publishers.community.slack.slack_layout import Summary, AttachRuleInfo \
+        , AttachPublication, AttachFullRecord
+
+@disable
+@rule(
+    logs=['fleet:results'],
+    publishers=[
```

Comment:
Mostly unrelated to the gist of your PR but had to chime in here: Publishers do not work on most of our default outputs. We wanted publishers to be mostly for human-readable presentation purposes, but force all default output stuff (for example, things that get dropped into Firehose-Athena) to have an unchanging format.

These publishers are slack-specific, so unless there's an `outputs=['slack:...']` then this will not do anything. If you choose to go with slack, I also suggest using the output-specific notation:

```
SLACK_PUBLISHER_BUNDLE = [
  Summary,
  AttachRuleInfo,
  AttachPublication,
  AttachFullRecord,
]

... 
@rule( 
  ...
  publishers={
    'slack:_descriptor_': SLACK_PUBLISHER_BUNDLE
  }
)
```

Reply:
Agreed I didn't want to include a slack output so removed this section completly